src/h5web/vis-packs/core/heatmap/ColorBar.tsx

```diff
@@ -3,14 +3,14 @@ import { useMeasure } from '@react-hookz/web';
 import { adaptedNumTicks, createAxisScale } from '../utils';
 import styles from './ColorBar.module.css';
 import { getInterpolator, getLinearGradient } from './utils';
-import type { ScaleType, Domain } from '../models';
-import type { ColorMap } from './models';
 import { formatBound } from '../../../utils';
+import type { Domain } from '../models';
+import type { ColorMap, ScaleParams } from './models';
 import Tick from '../shared/Tick';
 
 interface Props {
   domain: Domain;
-  scaleType: ScaleType;
+  scaleType: ScaleParams;
```

Comment:
Should we rename this prop to `scaleParams` ? Also in `HeatmapVis` and `HeatmapMesh` ?

Reply:
I like the prop being named `scaleType`, because even with the gamma scale, you're still passing a scale type -- you just pass something else along with it.

What about `VisScaleType` instead of `ScaleParams`?

Reply:
We could instead have `VisxScaleType` for `Linear | Log | Symlog | Sqrt` and have `ScaleType = VisxScaleType | 'gamma'` ?